### ISSUE TITLE
Remove nested columns

### ISSUE THREAD

**Issue by jongacnik** [jongacnik/gr8#9]
We can remove nested columns in favor of %-of-parent classes:

```css
.x1 { width: 100% }
.x2 { width: 50% }
.x3 { width: 33.333% }
.x4 { width: 25% }
.x5 { width: 20% }
.x6 { width: 16.666% }
.x7 { width: 14.285% }
.x8 { width: 12.5% }
.x9 { width: 11.111% }
.x10 { width: 10% }
.x11 { width: 9.09% }
.x12 { width: 8.333% }
```

Maybe they should be called `pop` lol.

**Comment by jongacnik** [comment]
Maybe better if the prefix is `s` for *slice*?

```css
.s1 { width: 100% }
.s2 { width: 50% }
.s3 { width: 33.333% }
.s4 { width: 25% }
.s5 { width: 20% }
.s6 { width: 16.666% }
.s7 { width: 14.285% }
.s8 { width: 12.5% }
.s9 { width: 11.111% }
.s10 { width: 10% }
.s11 { width: 9.09% }
.s12 { width: 8.333% }
```

**Comment by jongacnik** [comment]
resolved #17 

🍾
